tests/countries/test_algeria.py

```diff
@@ -0,0 +1,72 @@
+from holidays.countries.algeria import Algeria, DZ, DZA
+from tests.common import TestCase
+
+
+class TestAlgeria(TestCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass(Algeria)
+
+    def test_country_aliases(self):
+        self.assertCountryAliases(Algeria, DZ, DZA)
+
+    def test_2022(self):
+        self.assertHoliday("2022-01-01")
+        self.assertHoliday("2022-01-12")
+        self.assertHoliday("2022-05-01")
+        self.assertHoliday("2022-05-02")
+        self.assertHoliday("2022-05-03")
+        self.assertHoliday("2022-07-05")
+        self.assertHoliday("2022-07-09")
+        self.assertHoliday("2022-07-10")
+        self.assertHoliday("2022-11-01")
```

Comment:
The TestCase:assertHoliday was designed to accept multiple date-like strings:
```suggestion
        self.assertHoliday(
            "2022-01-01",
            "2022-01-12",
            "2022-05-01",
            "2022-05-02",
            "2022-05-03",
            "2022-07-05",
            "2022-07-09",
            "2022-07-10",
            "2022-11-01",
        )
```

Plese group them where possible.